packages/@glimmer/component/addon/index.ts

```diff
@@ -1,21 +1,12 @@
 import { DEBUG } from '@glimmer/env';
 import ApplicationInstance from '@ember/application/instance';
 import { setComponentManager } from '@ember/component';
-import { gte } from 'ember-compatibility-helpers';
 
 import GlimmerComponentManager from './-private/ember-component-manager';
-import GlimmerComponent from './-private/component';
+import _GlimmerComponent from './-private/component';
 import { setOwner } from '@ember/application';
 
-if (gte('3.8.0-beta.1')) {
-  setComponentManager((owner: ApplicationInstance) => {
-    return new GlimmerComponentManager(owner);
-  }, GlimmerComponent);
-} else {
-  setComponentManager('glimmer', GlimmerComponent);
```

Comment:
Can you make sure this PR is labeled as breaking, and add a small section (under the existing PR description) that explains exactly _what_ is breaking (I think this might be the only thing?).